### PR TITLE
Add a feature flag for fork-detection

### LINKED ISSUE #945: Add a feature-flag to disable fork-handling in Lorre
As per subject

### PR DESCRIPTION
**Pull request by ivanopagano**
Closes #945

### CODE REVIEW

File: conseil-lorre/src/main/resources/application.conf [hunk 1]
```diff
@@ -30,6 +30,14 @@ lorre {
     }
   ]
 
+  # feature flags: select if some functionality should be turned off
+  enabled-features {
+    block-rights-fetching-is-on: true
+    block-rights-fetching-is-on: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
+    fork-handling-is-on: true
+    fork-handling-is-on: ${?CONSEIL_LORRE_FORK_DETECTION_ENABLED}
```
Comment (piotrkosecki) [inline conseil-lorre/src/main/resources/application.conf:38]: can you add this to docker-compose in the comment?

Comment (ivanopagano) [reply conseil-lorre/src/main/resources/application.conf:38]: Sure, thanks

Comment (ivanopagano) [reply conseil-lorre/src/main/resources/application.conf:38]: Done that, is that needed anywhere else?

File: conseil-lorre/src/main/resources/application.conf [hunk 2]
```diff
@@ -30,6 +30,14 @@ lorre {
     }
   ]
 
+  # feature flags: select if some functionality should be turned off
+  enabled-features {
+    block-rights-fetching-is-on: true
+    block-rights-fetching-is-on: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
+    fork-handling-is-on: true
```
Comment (vishakh) [inline conseil-lorre/src/main/resources/application.conf:37]: Please set the default to false.

Comment (vishakh) [reply conseil-lorre/src/main/resources/application.conf:37]: On second thought, we cannot set this to false unless we also relax the db constraints.

Comment (ivanopagano) [reply conseil-lorre/src/main/resources/application.conf]: I'd say that the flag doesn't change much in terms of consistency guarantees for the DB... yet it removes additional sources of errors when the fork happens, until I solve the bug

**Review by vishakh — APPROVED** [review]

**Review by piotrkosecki — APPROVED** [review]

File: conseil-lorre/src/main/resources/application.conf (hunk 1 above)
Comment (piotrkosecki) [reply conseil-lorre/src/main/resources/application.conf:38]: no, I dont think so

**Pull request merged**